packages/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -20,81 +20,134 @@ import avatarImg from '../../Avatar/examples/avatarImg.svg';
 
 ### Basic
 
+Basic dropdowns present all selectable items in a menu upon clicking within the dropdown box. The `isOpen` property is used to flag a menu as opened. 
+
+All of the items in a dropdown list are created using the `DropdownItem` component. The following example shows a few different states of dropdown items. For each item, you may: 
+- Use the `tooltip` property to enable tooltip support. 
+- Use the `isDisabled` property to disable certain menu items.
+- Use the `isAriaDisabled` property to 
```

Comment:
Use the `isAriaDisabled` property to make a menu item appear disabled, while still being able to receive keyboard focus and trigger a tooltip to display